lib/modules/datasource/maven/util.ts

```diff
@@ -66,7 +66,7 @@ function isUnsupportedHostError(err: HttpError): boolean {
 
 const cacheProvider = new PackageHttpCacheProvider({
   namespace: 'datasource-maven:cache-provider',
-  ttlMinutes: 7 * 24 * 60,
+  ttlMinutes: 15,
```

Comment:
Can we cache-bust the existing?

Reply:
I agree, that we should cache bust this. Otherwise already cached dependencies are only reevaluated after 7 days of the update.

To bust the cache, we need to change the namespace, right?

Additionally, because the standard ttl for PackageHttpCacheProvider is 15min, we can drop the explicit number here (similar to how the npm cache does it): 

```suggestion
```

Reply:
We store timestamp of when data is cached and not when it expires, so if we change ttl parameter, the expiry date is recalculated and the request is made to the upstream.